irods/exception.py

```diff
@@ -69,7 +69,7 @@ class MultipleResultsFound(QueryException):
     pass
 
 
-class NotImplementedInIRODSServer(RuntimeError):
+class NotImplementedInIRODSServer(PycommandsException):
```

Comment:
Why change the base type?

Reply:
A number of Python iRODS Client exceptions are derived from it, in particular `OperationNotSupported`. My own theory is that the PRC was originally supposed to support  a substitute for, or customization of, iCommands, and that a PycommandsException meant you were attempting something that was unsupported.  I'd be equally happy reverting the `NotImplementedInIRODSServer` base class back to `RuntimeError`, changing it to `Exception`, or giving it the same base class as `NotImplementedInIRODSServer` (which was the motivation for this change).

@alanking and i discussed this earlier.  the advantage to this approach might be that an application developer could use `PycommandsException` as a catch-all for both `NotImplementedInIRODSServer` and `OperationNotSupported`.

Reply:
Ok. Can this break existing code?

Reply:
`NotImplementedInIRODSServer` was introduced in commit https://github.com/irods/python-irodsclient/commit/3015e40a1c1c8868ec22efe35751e3d0d85bebd8 which is not in a release.

Reply:
Right, otherwise I wouldn't have considered this change without a lot more todo.  (Some doc changes at least)

Reply:
Cool. Carry on.